channeldb/graph.go

```diff
@@ -236,6 +248,44 @@ func (c *ChannelGraph) ForEachChannel(cb func(*ChannelEdgeInfo, *ChannelEdgePoli
 	})
 }
 
+// DisabledChannelIDs returns the channel ids of disabled channels.
+// A channel is disabled when two of the associated ChanelEdgePolicies
+// have their disabled bit on.
+func (c *ChannelGraph) DisabledChannelIDs() ([]uint64, error) {
+
+	var disabledChanIDs []uint64
+	chanEdgeFound := make(map[uint64]bool)
```

Comment:
looks like this only needs to be a set of the channel ids, so we can make this `map[uint64]struct{}`

Reply:
yes, that's better. changed.